test/e2e/e2ef_test.go

```diff
@@ -34,6 +34,7 @@ const (
 	nvmepool     = "opensds-volumes-nvme"
 	defaultgroup = "opensds-volumes-default"
 	localIqn     = "iqn.2017-10.io.opensds:volume:00000001"
+	iscsi        = "iscsi"
```

Comment:
I prefer to use `iscsiProtocol` here, also need to add `nvmeofProtocol`.

Reply:
Done.